opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/HeaderNames.java

```diff
@@ -16,6 +16,7 @@
     public static final String FINTECH_USER_ID = "Fintech-User-ID";
     public static final String SERVICE_SESSION_PASSWORD = "Service-Session-Password";
     public static final String X_XSRF_TOKEN = "X-XSRF-TOKEN";
+    public static final String TPP_REDIRECT_PREFERRED = "TPP-Redirect-Preferred";
```

Comment:
The tests won't work. We don't allow passing this through API. See 
```
@Service
public class TppRedirectPreferredResolver {

    public Boolean isRedirectApproachPreferred(CurrentBankProfile config) {
        return config.isTryToUsePreferredApproach()
                       ? config.getPreferredApproach() == Approach.REDIRECT
                       : null;
    }
}
```
You need to alter bank configuration to enable this header

Reply:
Was considered, tests changed